### PR TITLE
Run TypeScript tests

### PR DESCRIPTION
**Pull request by dbkr**
There aren't any yet, but I'm about to write a test and it's going
to be in TypeScript and I don't want to lump this change in with
a bunch of other stuff.

### CODE REVIEW

File: src/settings/Settings.ts [hunk 1]
```diff
@@ -398,6 +398,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
         default: null,
     },
+    "voip_mxid_translate_pattern": {
+        supportedLevels: LEVELS_UI_FEATURE, // not a UI feature, but same level (config only, maybe .well-known in future)
+        default: null,
+    },
```
Comment (t3chguy) [inline src/settings/Settings.ts:404]: this looks unrelated

**Comment by t3chguy** [comment]
Funnily enough I was about to do the same thing!!

image

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/settings/Settings.ts (hunk 1 above)
Comment (dbkr) [reply src/settings/Settings.ts]: oops. Fixed.

**Comment by dbkr** [comment]
Hah. I'm genuinely ashamed of the amount of time I spent trying to figure out why the test I just write was not being run. 😢

**Comment by t3chguy** [comment]
I had a slightly lighter experience, I clicked my IDE's run just this test suite button and the console said "No tests found" which kinda helped me realise what was happening :P
